power-hour.py

```diff
@@ -22,7 +22,8 @@ def call_yt_dl(yt_url):
     '''downloads m4a to working dir'''
     args = [
         'youtube-dl',
-        '-f', '140',  # m4a audio track
+        '-f', 'best[height<=720]', 
```

Comment:
not sure why, but this degrades audio quality from ~125 Kb/s to ~95 Kb/s

Reply:
Fixed that. I now also use opus as audio codec.